app/serializers/allocation_serializer.rb

```diff
@@ -5,6 +5,11 @@ class AllocationSerializer < ActiveModel::Serializer
 
   has_one :from_location
   has_one :to_location
+  has_many :moves
```

Comment:
When we include moves in allocations, we are currently introducing circular nested dependencies, this is because we also have an association in the `MoveSerializer` :
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/serializers/move_serializer.rb#L13
This means that any move in the `included` section will have allocation defined in the relationships, even though we are seeing it under the allocation endpoint itself. This seems to be in conflict with the JSON:API specs as from the examples I've seen we don't usually return the relationship again if a model is included.

We currently have precedence of this in the codebase (around court hearings I believe). I could introduce either separate serialisers for moves to avoid this (inherit from a base and not include the `belongs_to` etc.), or keep it as is for now and add a ticket to tackle inconsistencies in the JSON:API spec epic. I have chosen the latter but would like to hear people's thoughts on this cc. @teneightfive